libs/core/pyproject.toml

```diff
@@ -86,6 +86,7 @@ ignore = [
     "FIX002", # Line contains TODO
     "ISC001", # Messes with the formatter
     "PERF203", # Rarely useful
+    "PLC0414",  # Enable re-export
```

Comment:
OK change. The rule doesn't make sense as re-exporting with an alias is actually the recommended way to re-export